docs/HACKING.md

```diff
@@ -97,7 +97,7 @@ were made to the files in config/.
 If the operator version needs to be changed, the `WMCO_VERSION` variable at the top of the repo's Makefile should
 be set to the new version.
 
-If the bundle channel needs to be changed, the `CHANNELS` and/or `DEFAULT_CHANNELS` variable within the the Makefile
+If the bundle channel needs to be changed, the `CHANNELS` and/or `DEFAULT_CHANNELS` variable within the Makefile
```

Comment:
Both these changes have nothing to do with this PR. Please revert. This is a general rule.

Reply:
[PR opened](https://github.com/openshift/windows-machine-config-operator/pull/483)